packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -272,9 +272,36 @@ export const OAuthLoginPage = () => {
 
   return (
     <ContentWrapper>
-      <div className={cn(styles.centeredContent, styles.titleContainer)}>
-        <h1 className={styles.title}>{titleText}</h1>
-      </div>
+      <Flex alignItems='center' direction='column'>
+        <Flex gap='l' alignItems='center' mb='l'>
+          <Flex h='88px' w='88px'>
+            <img src={AppIcon} alt={'Audius Logo'} />
+          </Flex>
+          <IconTransaction color='default' />
+          <Flex h='88px' w='88px'>
+            {appImage ? (
+              <img src={appImage} alt={`${appName} Image`} />
+            ) : (
+              <Flex
+                w='100%'
+                justifyContent='center'
+                alignItems='center'
+                borderRadius='l'
+                css={{ backgroundColor: 'var(--harmony-n-200)' }}
```

Comment:
I think this is what `Paper` is for? Probably shouldn't be setting background colors like this.

Reply:
it's a again a weird icon component rather than a dialog or pop up and the background color is outside of our semantic values. will merge this but chat w julian some